tutor4points/core/forms.py

```diff
@@ -142,6 +142,34 @@ class Meta:
     helper = FormHelper()
     helper.form_id = 'form'
 
+class RateTutorForm(forms.ModelForm):
+
+    class Meta:
+        model = Rating
+
+        # layout where want fields to be, type in order you want it to appear
+        fields = ('rating', 'comment','rating_type')
+
+        # customize placeholders
+        widgets = {
+            'rating':
+            forms.NumberInput(attrs={'placeholder': '1-5'}),
+            'comment':
+            forms.TextInput(attrs={'placeholder': 'comment here'}),
+            'Type of rating (tutor or tutee)':
+            forms.TextInput(attrs={'placeholder': 'comment'}),
+
+        }
+
+        # customize form labels
+        labels = {
+            'rating': "Enter a rating 1-5 stars",
+            'comment': "Explain how you're session went",
+            'rating_type': "Is the user you are rating a tutor or tutee?",
```

Comment:
It would be helpful if the app automatically chooses this field for the user because it might be confusing for the user. The Tutee/tutor can be a hidden field in the form and filled out based on whether the user checked off a tutor/tutee session. You can probably discuss with @dtuckersuh how you guys can make that possible on the "View accepted tutor requests" page. 

One way is just to name the "div" and then check if the button belongs to a tutor or a tutee div. 

Another way is to have a hidden input field and pass in the request-id along with the form. You can then get the request instance from the model and check if the user is a tutor or tutee. Daniel might be able to help with the scripting part, or you could look at the "View Tutor Request/Request Tutor" page for the scripting.

Reply:
You might be able to just not display this field in the form, and in the backend/view, you could retrieve the tutee/tutor status and set the field there. That might be simpler that passing the information to the front end and then sending it back as hidden input.